### PR TITLE
Update snapdragon to 0.11

### PR DESCRIPTION
**Pull request by danez**
## Description

This update snapdragon to the latest version 0.11

## Checklist

- [x] Breaking change (Needs to be major version as otherwise micromatch picks it up, which uses still an older version of snapdragon)
- [x] Test pass (not yet see here-be/snapdragon#15, they do with the snapdragon fix locally)

### CODE REVIEW

**Comment by danez** [comment]
Okay i updated snapdragon and seems everything works now

File: lib/utils.js [hunk 1]
```diff
@@ -59,6 +60,7 @@ utils.instantiate = function(ast, options) {
   } else {
     snapdragon = new Snapdragon(options);
   }
+  snapdragon.use(capture())
```
Comment (danez) [inline lib/utils.js]: Should I only add capture when doing new Snapdragon() here maybe? But I guess using `snapdragon.use(capture())` multiple times on the same snap instance does not harm anything.

Comment (jonschlinkert) [reply lib/utils.js]: I'll leave that up to you. I can go either way, and you have thought it though.

**Comment by jonschlinkert** [comment]
this LGTM!

**Pull request merged**